src/content/getusermedia/record/js/main.js

```diff
@@ -89,24 +81,29 @@ function toggleRecording() {
 }
 
 function startRecording() {
+  var options = {mimeType: 'video/vp9'};
```

Comment:
Does this mean VP9 will always be used? Will it fall back to VP8 if VP9 is not supported?